README.md

```diff
@@ -169,7 +169,7 @@ sudo GOPATH=$GOPATH SYSTEMD_NSPAWN_PATH=<path_to_your_nspawn_binary> DOCKER_API_
 - [x] bootstrapping nodes
 - [x] initialize node-0 as master with `kubeadm`
 - [x] join nodes to form a cluster
-- [ ] create deployments
+- [ ] create deployments (#14)
```

Comment:
Github seems to generate links automatically in comments, but not in a markdown file, unfortunately...